### PR TITLE
Fix kalman stuff and clean up

### PR DESCRIPTION
**Pull request by haraschax**
Hey rwschubert,

I saw you fixed up some stuff in Laika, if at some point you want to try merge in the fixes to the kalman filter and other improvements that would be greatly appreciated, thanks!

### CODE REVIEW

**Comment by rwschubert** [comment]
Thank you for contacting me! Yes, I can try.
Honestly, I have mostly merged the current version of the Laika library from the openpilot project to make it a standalone version supporting the latest release with Python 3.

(Furthermore, I added a lot of comments since I found it quite hard to follow...just my opinion.)

File: run_docker.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Build image
+docker build -t rwschubert/kalman-laika .
```
Comment (AllWashedOut) [inline run_docker.sh:2]: Is it intended that this file contains "**rwschubert**/kalman-laika", on line 2 and line 13? Just wondering if that's an artifact from your workspace or something.

Comment (rwschubert) [reply run_docker.sh:2]: It is just the tag of the Docker image that is chosen when the Docker image is built.
By specifying it with `-t` you could set another tag.
